realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -426,6 +427,68 @@ public SyncUser run() {
         }.start();
     }
 
+    /**
+     * Helper method for Admin users in order to look up a {@code SyncUser} using the identity provider and the used username.
+     * @param provider identity providers {@link io.realm.SyncCredentials.IdentityProvider}.
+     * @param providerId username or email used to create the account first time.
+     * @return {@code SyncUser} associated with the given identity provider and providerId.
+     * @throws ObjectServerError if the user could not be found.
+     */
+    public SyncUser retrieveUser(final String provider, final String providerId) throws ObjectServerError {
```

Comment:
I agree, as long as we align with other bindings ... the original proposed name was `retrieveUser` https://github.com/realm/realm-java/issues/4828